pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -524,165 +524,6 @@ func DagFromState(state PipelineRunState) (*dag.Graph, error) {
 }
 
 func TestGetNextTasks(t *testing.T) {
-	tcs := []struct {
-		name         string
-		state        PipelineRunState
-		candidates   map[string]struct{}
-		expectedNext []*ResolvedPipelineRunTask
-	}{{
-		name:         "no-tasks-started-no-candidates",
```

Comment:
Many of these use cases used an impossible state / candidates combination.
Now that candidates are calculated within `GetNextTasks` it's not possible anymore to force such impossible scenarios.